src/tools/clippy/clippy_lints/src/redundant_clone.rs

```diff
@@ -134,6 +134,7 @@ impl<'tcx> LateLintPass<'tcx> for RedundantClone {
             }
 
             if let ty::Adt(def, _) = arg_ty.kind() {
+                // FIXME: This is not correct with `#[manually_drop]`, as that is just like any other type.
```

Comment:
Should this instead be changed to check against the lang_item DefId then?

Reply:
We could do that, but then we're committing to ManuallyDrop remaining special, right? (And this e.g. prohibits removing the lang item)